talkitover_app.py

```diff
@@ -574,6 +605,9 @@ def choose_bot_wordy_response(message, clientId):
                             "i have difficulties with my family", "i'm having difficulties with my family", "im having difficulties with my family", "ive got difficulties with my family", \
                             "i have family worries", "i'm having family worries", "im having family worries", "ive got family worries", "ive got worries with my family", \
                             "i have worries with my family", "i'm having worries with my family", "im having worries with my family", "ive got worries with my family"]
+    iWantAFriendArray = ["i want a friend", "i need a friend", "I have no friends", "I don't have any friends", "I havent got any friends", "ive got no friends" "I don’t have friends"]
+    iDontSeeManyPeopleArray = ["i dont see many people", "i dont see a lot of people"]
+    helpArray = ["Help", "Help me", "I need help"]
```

Comment:
Again, I think the capital letter is stopping the code from delivering a response.